luigi/contrib/sqla.py

```diff
@@ -141,8 +141,14 @@ def requires(self):
 import itertools
 import logging
 import luigi
-import sqlalchemy
 import os
+import warnings
+
+try:
+    import sqlalchemy
+except ImportError:
+    # Don't fail on import because we want the documentation to be generated
+    warnings.warn("sqlalchemy could not be imported, db_task_history will not work", warnings.ImportWarning, stacklevel=3)

```

Comment:
sphinx-apidoc needs to be able to import the module to generate docs. Currently `luigi.contrib.sqla` throws an exception at the module level and that's why no documentation is generated: http://luigi.readthedocs.org/en/latest/api/luigi.contrib.sqla.html